cli/cmd/tap.go

```diff
@@ -31,6 +35,51 @@ type tapOptions struct {
 	output      string
 }
 
+type endpoint struct {
+	IP       string            `json:"ip"`
+	Port     uint32            `json:"port"`
+	Metadata map[string]string `json:"metadata"`
+}
+
+type streamID struct {
+	Base   uint32 `json:"base"`
+	Stream uint64 `json:"stream"`
+}
+
+type requestInitEvent struct {
+	ID        *streamID      `json:"id"`
+	Method    *pb.HttpMethod `json:"method"`
```

Comment:
this struct is an enum and doesnt render well.  lets make this a string field like "GET" or "POST"